### ISSUE TITLE
Request review of updated contact details for security.

### ISSUE THREAD

**Issue by warmanaMOJ** [ministryofjustice/security-guidance#340]
Hello @L-Crosby or @cybersquirrel ,

Please may I request a review of updates to the contact information within the published policy and guidance? These updates are intended to align better with current organisational processes.

Specifically, we have consolidated the contact details so that advice to contact Cyber Consulting or Operational Security now always refers to security@... as a first point of contact. A good example is in the Acceptable Use Policy.

The current [published policy statement `POL.ITAUP.007`](https://security-guidance.service.justice.gov.uk/acceptable-use-policy/#protection-of-assets) states that "Any request to perform an action on an IT system which contravenes its SyOPs **SHALL** be approved by the [Cyber Assistance Team](mailto:CyberConsultancy@digital.justice.gov.uk) and the [Operational Security Team](mailto:OperationalSecurityTeam@justice.gov.uk), or the MoJ Chief Information Security Officer (CISO), before the action is taken.

In the [draft update](https://github.com/ministryofjustice/security-guidance/blob/links0411/docs/acceptable-use-policy.md#protection-of-assets), this has been consolidated to: "Any request to perform an action on an IT system which contravenes its SyOPs **SHALL** be approved by the [Security team](mailto:security@justice.gov.uk), or the MoJ Chief Information Security Officer (CISO), before the action is taken."

In summary, this update replaces all references to Cyber Consulting or Operational Security teams with a single, unified reference to the Security team, using email address security@...

N.B. While the changes themselves are straightforward and small, there are a lot of them. Most of the content pages are affected; over 170 files have being changed.

A full set of the changes introduced in this release may be seen [here](https://github.com/ministryofjustice/security-guidance/compare/main...links0411).

Many thanks.

**Comment by cybersquirrel** [comment]
Thanks @warmanaMOJ  - these changes all seem sensible to me. I am wondering if we need the "...or by the CISO" in the policy statement, as that seems to invite people to contact me directly, rather than through the security@ where we can keep a log of decisions? Not a blocker to publication though,

**Comment by warmanaMOJ** [comment]
@cybersquirrel 

Thank you - I've updated four files to remove '...or by the CISO...' in favour of contacting initially through security@..., as suggested.

The affected files (with links to updated previews) are:

- [Acceptable use policy](https://github.com/ministryofjustice/security-guidance/blob/links0411/docs/acceptable-use-policy.md)
- [Data Handling and Information Sharing Guide](https://github.com/ministryofjustice/security-guidance/blob/links0411/docs/data-handling-and-information-sharing-guide.md)
- [Email blocking policy](https://github.com/ministryofjustice/security-guidance/blob/links0411/docs/email-blocklist-policy.md)
- [Policies for MacBook Administrators](https://github.com/ministryofjustice/security-guidance/blob/links0411/docs/policies-for-macbook-administrators.md)

I'll proceed with the publication approval process - but please let me know if these four changes need further work.

Many thanks.

**Comment by warmanaMOJ** [comment]
Thank you, [published](https://security-guidance.service.justice.gov.uk/#cyber-and-technical-security-guidance).
